server/controllers/findUnallocatedCasesController.ts

```diff
@@ -111,6 +111,13 @@ export default class FindUnallocatedCasesController {
     // eslint-disable-next-line security-node/detect-dangerous-redirects
     res.redirect(`/probationDeliveryUnit/${pduCode}/find-unallocated`)
   }
+
+  async clearFindUnallocatedCases(req: Request, res: Response, pduCode: string): Promise<void> {
+    const { token, username } = res.locals.user
+    await this.userPreferenceService.clearAllocationDemandPreference(token, username)
+    // eslint-disable-next-line security-node/detect-dangerous-redirects
+    res.redirect(`/probationDeliveryUnit/${pduCode}/find-unallocated`)
```

Comment:
I have just realised...  am confused by this `pduCode` - is it not the regionCode?

Reply:
it is pduCode, I tried to make it region code but then the manage my teams link breaks